src/Widgets/PaymentPlans/PaymentPlans.module.css

```diff
@@ -95,3 +100,9 @@
 .unClickable {
   cursor: not-allowed;
 }
+
+.oneLine {
```

Comment:
It's supposed to be condensedView now (same for above)

Reply:
as you want :) i made the changes